microcosm_pubsub/registry.py

```diff
@@ -77,11 +77,11 @@ def find(self, media_type):
         except KeyError:
             # use convention otherwise
             if self.lifecycle_change.Deleted in media_type.split("."):
-                schema = IdentityMessageSchema(media_type)
+                schema = IdentityMessageSchema()
             elif self.lifecycle_change.Changed in media_type.split("."):
-                schema = ChangedURIMessageSchema(media_type)
+                schema = ChangedURIMessageSchema()
             else:
-                schema = URIMessageSchema(media_type)
+                schema = URIMessageSchema()
```

Comment:
By this point we have found the schema we're going to be using for the message, based on the media type. This means that we won't need the media type again.